### DISCUSSION TITLE
Export imported osu!stable scores from osu!lazer?

### DISCUSSION THREAD

**Discussion by fopsykl** [ppy/osu#21195] [Q&A] [answered]
Hi,

Is there a way to export any osu!stable scores that have been imported into osu!lazer? I've reinstalled stable recently and didn't backup any `.db` file containing scores, collections, or beatmap info. I have imported both collections and scores into lazer a couple of days prior to reinstalling stable and was wondering if there is a way to export those scores and collections back into stable? 

**(I have managed to get my collections back from `.../AppData/Roaming/osu/collections.db.migrated`)**

### imported osu!stable scores in osu!lazer (scores with no avatar were imported before a certain update earlier this year)
![imported osu!stable scores in osu!lazer](https://user-images.githubusercontent.com/56948009/201016028-a0e50999-36cc-482b-bf8c-4ebcb743976e.jpg)

### same map in osu!stable after reinstall
![same map in osu!stable after reinstall](https://user-images.githubusercontent.com/56948009/201018054-595dff26-ad16-478a-9f65-509e1b17aadd.jpg)

(sorry for the million updates, this is the first discussion I've posted and just wanted to make it understandable)

Thanks,
Ben :D

**Comment by peppy** [comment, marked as answer]
You should be able to right click on a score to export it, but there's no way to mass-export scores.
